ivy/functional/backends/jax/statistical.py

```diff
@@ -135,6 +135,29 @@ def var(
     )
 
 
+def cov(
+    x: JaxArray,
+    /,
+    *,
+    y: Optional[JaxArray],
+    rowvar: Optional[bool] = True,
+    bias: Optional[bool] = False,
+    ddof: Optional[int] = None,
+    fweights: Optional[Union[int, Sequence[int]]],
+    aweights: Union[Sequence[int], Sequence[float]],
+) -> JaxArray:
+    ret = jnp.cov(
```

Comment:
As it is the frontend of a specific framework, which is `jax` in this case, the implementation should be only in `ivy`, which means that we can only use Ivy functions.

Reply:
No, previously you understood right! At first, i intended to implement the frontend cov function for jax, but i saw that there was no backend function for it. Then, i track the dependencies and built for my first pull the backend function for it. Late, after pr accepted and merged, i can implement the ivy "middle end" and then the respective frontend function.

Reply:
@YasCoMa , This PR of yours is linked to a frontend issue `cov`, also the backend function does exist, you might me looking in the `statistical` submodule :slightly_smiling_face: 
kindly take a look at this https://github.com/unifyai/ivy/blob/27772879639877c2f1553ae5b5dfbebc8cfc71fe/ivy/functional/backends/jax/experimental/linear_algebra.py#L153

Reply:
Thanks, i really did not check the experimental section before creating a new one. I removed my implementation from the statistical in functional. And i implemented nanmin and nanmax that were not checked in the issue neither in the experimental/statistical